repos/tap/actions/files/local/setActiveFile.js

```diff
@@ -24,12 +24,9 @@ const getScreen = screenId => {
  */
 export const setActiveFile = (fileModel, content, screenId) => {
   const screenModel = getScreen(screenId)
-
-  // If no screen is found, then we can't set the file
-  // So log an error and return
   if(!screenModel) return devLog(`error`, `Can not find screen from id`, screenId)
-  
-  const updatedFile = { ...fileModel, modified: content || fileModel.modified }
+  const updatedFile = fileModel
```

Comment:
Anytime we update something from the store, we should create a version of it
* So this should look like this => `const updatedFile = { ...fileModel }`

Reply:
ah ok. otherwise it modifies the original content before it gets dispatch. didnt notice that.
updated

Reply:
Yah, exactly